packages/domain/lib/base/mappers/mapper_people_model.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:domain/base/mappers/base_mappers/mapper_base_two_in.dart';
+import 'package:domain/const/configuration.dart';
+import 'package:domain/entity/cast_response/cast_response.dart';
+import 'package:domain/entity/cast_response/tmdb_people_response.dart';
+import 'package:domain/model/response_model_people.dart';
+
+class MapperPeopleModel extends MapperTwoIn<List<Cast>,
```

Comment:
naming - MapperPeopleModel -> PeopleModelMapper. (the same as HomeBloc, not BlocHome)